oneanddone/base/templatetags/jinja_helpers.py

```diff
@@ -0,0 +1,116 @@
+import datetime
```

Comment:
Please feel free to upstream pull request things that you believe are broadly useful  to sugardough. I think `fe` is one of these things.